index.html

```diff
@@ -0,0 +1,152 @@
+            This peaceful gathering of different cultures for the common cause of music, celebration and entertainment is what the Island of Freedom is all about.
+          </p>
+        </div>
+        <div class="card-b col-12-xs col-12-sm col-5-md col-3-lg col-2-xl">
+          <h3 class="card-title">
+            Art and Inspiration
+          </h3>
+          <p class="">
+            While music is an important part of Sziget, art is also a core element. There are multiple venues including a circus, theater, museum quarter and much more. Artists and performers come 
+            from all over the world to display their talents. There are also opportunities for Szitizens and artists to work together in order to create memorable and meaningful artwork.
+          </p>
+        </div>
+        <div class="card-b col-12-xs col-12-sm col-5-md col-3-lg col-2-xl">
+          <h3 class="card-title">
+            Wherever you go
+          </h3>
+          <p class="">
+            Roads give you chances to seek and explore. Want an adventure? Just open your door. Award-winning author Pat Zietlow Miller's lilting rhymes celebrate the possibilities that 
+            lie beyond the next bend in the road--the same road that will always lead you home again. Look back at Sziget 2019, where these words were brought to life on the Island of Freedom!
+          </p>
+        </div>
+        <div class="card-b col-12-xs col-12-sm col-5-md col-3-lg col-2-xl">
+          <h3 class="card-title">
+            Join the Love Revolution
+          </h3>
+          <p class="">
+            We believe that the power of diversity is a unifying force that can help create change around us. We come in all shades, from different backgrounds, 
+            but no matter what we are a community who share common values.
+          </p>
+        </div>
+        <div class="card-b col-12-xs col-12-sm col-5-md col-3-lg col-2-xl">
+          <h3 class="card-title">
+            Our cause: SUPERAR
+          </h3>
+          <p class="" id="ticketslink">
+            Sziget partnered with Superar this year, to support the program’s initiative on spreading music to children coming from underprivileged backgrounds. 
+            Music is a powerful tool that can help children improve their focus, motivation, creativity and confidence, while allowing them to overcome socio-economic boundaries. 
+            Every reusable cup dropped off at a collection point at Sziget, went towards supporting the Superar program.
+          </p>
+        </div>
+      </div>
+      <button type="button" class="btn-red btn-or-b mt-2" id="featured">Buy your ticket HERE!</button>
+    </div>
+    <div class="featured-artists pb-2">
+      <h2 class="h-secondary pt-2">Featured Artists</h2>
+      <div class="separator"></div>
+      <div class="row" id="artistsholder"></div>
+    </div>
```

Comment:
- [ ] This section is almost like the template, you need some updates to add the sub-title and the small divider, and the background for the profile images, and it should be ready

your design : 
![image](https://user-images.githubusercontent.com/89282221/190517084-11671df3-efb6-4f69-843d-e92c6c684032.png)


template design : 
![image](https://user-images.githubusercontent.com/89282221/190517192-90084d69-054b-4a3e-ae55-497918877350.png)